supervisor/api/proxy.py

```diff
@@ -122,10 +122,10 @@ async def _websocket_client(self):
 
             if data.get("type") != "auth_required":
                 # Invalid protocol
-                _LOGGER.error(
-                    "Got unexpected response from Home Assistant WebSocket: %s", data
+                raise APIError(
+                    "Got unexpected response from Home Assistant WebSocket: {data}",
```

Comment:
```suggestion
                    f"Got unexpected response from Home Assistant WebSocket: {data}",
```
The string contains a variable and needs to be prefixed with `f`